### PR TITLE
add more COL types to testnet.json and plugin, set correct decimal pl…

### PR DESCRIPTION
**Pull request by b-pmcg**
…ace for COL5

### CODE REVIEW

File: src/eth/EthereumTokenService.js [hunk 1]
```diff
@@ -75,6 +75,10 @@ export default class EthereumTokenService extends PrivateService {
       return new PethToken(contract, this.get('web3'), tub);
     }
 
+    if (symbol === tokens.COL5) {
```
Comment (tyler17) [inline src/eth/EthereumTokenService.js]: I don't think tokens.COL5 is defined

Comment (levity) [reply src/eth/EthereumTokenService.js]: We can’t hardcode this here—it should rely on the `decimals` value from the config

Comment (b-pmcg) [reply src/eth/EthereumTokenService.js]: Removed this and added it to the tokens config in dai-plugin-config https://github.com/makerdao/dai-plugin-config/blob/master/src/utils/configHelper.js#L52

**Comment by b-pmcg** [comment]
Closing this because the handling the decimals has been moved to the dai-plugin-config, and the rest of this PR overlaps with the work that @abylaw and @ethanbennett are doing.

**Pull request closed without merging**